.github/SECURITY.md

```diff
@@ -1,5 +1,12 @@
 # Security Vulnerabilities
```

Comment:
@KirkMunro, your last commit had 2 failures in `PowerShell-CI-windows`
PowerShellGet - Script tests (Admin).Should install a script correctly to the required location with AllUsers scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PowerShellGet\PowerShellGet.Tests.ps1: line 234
234:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```
PowerShellGet - Script tests.Should install a script correctly to the required location with default CurrentUser scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PowerShellGet\PowerShellGet.Tests.ps1: line 207
207:         $installedScriptInfo | Should -Not -BeNullOrEmpty
```